config/initializers/assets.rb

```diff
@@ -17,4 +17,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( accessible-autocomplete.min.js )
```

Comment:
Don't think this is required, as the file should already be in the right place to be picked up by the asset pipeline?